tests/genesis/genesis_test.go

```diff
@@ -416,8 +425,11 @@ func validateNodes(t *testing.T, genesis *registryAPI.Genesis, source *oasis.Con
 			id, entity_id, expiration,
 			tls_pubkey, tls_next_pubkey, p2p_pubkey,
 			vrf_pubkey, roles, software_version
-		FROM %s.nodes_checkpoint
-		WHERE roles LIKE '%s'`, chainID, "%validator%"),
+		FROM
+			%s.nodes_checkpoint
+		WHERE
+			roles LIKE '%%validator%%'
```

Comment:
Does the double percent sign add anything? or should it be 
```suggestion
			roles LIKE '%validator%'
```

Reply:
The `%%` inside a string just resolves to a single literal %. The special escape sequence is needed to differentiate the literal % from those % that are used for formatting, e.g. %s.
The previous code used `"%validator%"` without `%%`, and I guess because that string was not passed into `Sprintf` or similar, go decided not to complain. So both solutions work, this edit is just a tad cleaner.